src/extension/app.css

```
@@ -899,6 +899,22 @@
   content: "Please wait...";
 }
 
+.hlx-sk-overlay .modal.modal-preview-onedrive::before {
+  content: "Pro tip: Press Control + S before previewing to see the latest changes immediately...";
+}
+
+.hlx-sk-overlay .modal.modal-preview-onedrive-mac::before {
+  content: "Pro tip: Press Command + S before previewing to see the latest changes immediately...";
```

```suggestion
  content: "Pro tip: Press ⌘ + S before previewing to see the latest changes immediately...";
```

(or `&#8984;`, in case the ⌘ doesn't work.